Assets/LeapMotionModules/ElementRenderer/Scripts/Features/Tint/LeapGuiTintFeature.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+[AddComponentMenu("")]
+[LeapGuiTag("Tint")]
+public class LeapGuiTintFeature : LeapGuiFeature<LeapGuiTintData> {
+  public const string FEATURE_NAME = LeapGui.FEATURE_PREFIX + "TINTING";
+
+#if UNITY_EDITOR
+  public override void DrawFeatureEditor(Rect rect, bool isActive, bool isFocused) { }
+
+  public override float GetEditorHeight() {
+    return 0;
```

Comment:
Ah, it wasn't obvious to me that the features here magically propagated to the elements (where they do have a Tint Color parameter).

Hmm, probably my first major stumble with the architecture of the system and how features are enabled or disabled...